api/admin/cli/main.go

```diff
@@ -0,0 +1,311 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/base32"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/wissance/Ferrum/managers/redis"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/wissance/Ferrum/api/admin/cli/operations"
+	"github.com/wissance/Ferrum/data"
+	"github.com/wissance/Ferrum/logging"
+	sf "github.com/wissance/stringFormatter"
+)
+
+type Manager interface {
+	GetRealm(realmName string) (*data.Realm, error)
+	GetClient(realmName string, clientName string) (*data.Client, error)
+	GetUser(realmName string, userName string) (data.User, error)
+
+	CreateRealm(newRealm data.Realm) error
+	CreateClient(realmName string, clientNew data.Client) error
+	CreateUser(realmName string, userNew data.User) error
+
+	DeleteRealm(realmName string) error
+	DeleteClient(realmName string, clientName string) error
+	DeleteUser(realmName string, userName string) error
+
+	UpdateRealm(realmName string, realmNew data.Realm) error
+	UpdateClient(realmName string, clientName string, clientNew data.Client) error
+	UpdateUser(realmName string, userName string, userNew data.User) error
+
+	SetPassword(realmName string, userName string, password string) error
+}
+
+const defaultConfig = "./config.json"
+
+var (
+	argConfigFile  = flag.String("config", "", "Path relative to the application binary file")
```

Comment:
We should also pass config file as param (means ferrum config)

Reply:
Works as expected. The path to the configuration file is specified relative to the `ferrum-admin.exe` binary file.

Reply:
Ferrum allows to pass config and use different configs for different runs, therefore we should n't bind to single config